### ISSUE TITLE
Issues Connecting Mobile Devices to Accessibility Insights For Android

### ISSUE THREAD

**Issue by daphneyyt** [microsoft/accessibility-insights-for-android-service#167]
Dear Accessibility Insights for Android team,

My team and I are trying to conduct WCAG checks for a couple of mobile apps. However, Accessibility Insights For Android's system cannot detect our devices.

This is not the first time we are using Accessibility Insights for Android. We were able to run WCAG checks with the software in 2020 successfully. So, we are not exactly sure what is the problem.

We have followed all the steps listed in this URL: https://accessibilityinsights.io/docs/android/getstarted/setup/#connect-to-your-android-device

However, the issue persists.

Other steps we had taken, to no avail:
1) uninstall and install Accessibility Insights for Android

2) Reset developer mode settings on our Android devices

3) Updated the OS for the android devices

4) Restart the PC / restart the mobile devices

5) Reconnected the device multiple times

6)  Redownloaded the ABD file

For reference, this is the screen we are stuck on: (Selecting detect my device does nothing)
![AIFA1](https://user-images.githubusercontent.com/75086091/177471281-1846dd98-5055-4304-8ac2-10edc0766f16.png)

We appreciate your advice on any possible fixes we can take to proceed from here. We will be happy to provide more information if required. Thank you!

**Comment by dbjorge** [comment]
Hi @daphneyyt,

We'd be happy to help debug the issue you're seeing. We'll need a little more information to try to reproduce the problem:

* What is the specific device model and Android version you're trying to connect to?
* Does the issue persist if you attempt to connect to an emulator instead of a physical device?
* If you run `adb devices` from a command prompt, does it list the devices?

Finally, it would be helpful if you could share debug console output from Accessibility Insights for Android, which you can see if you run the program from a command line instead of a desktop shortcut. To do this (assuming you're running on Windows and have installed in the default location):

1. Open a Command Prompt
2. Run `"%LOCALAPPDATA%\Programs\Accessibility Insights for Android\Accessibility Insights for Android.exe"`
3. Progress as far as you can through the "Connect your device" prompts
4. Close the application and copy/paste the output in the command prompt
